cmd/up/activate.go

```diff
@@ -270,6 +270,7 @@ func (up *upContext) createDevContainer(ctx context.Context, app apps.App, creat
 				return err
 			}
 		} else {
+			delete(tList[name].App.ObjectMeta().Annotations, model.DeploymentAnnotation)
```

Comment:
Kubernetes can take time to update the `model.DeploymentAnnotation`. By deleting the annotation in the update, we can make sure when the new revision is available